src/Ledger/GovernanceActions.lagda

```diff
@@ -143,8 +140,6 @@ threshold pp ccThreshold' = λ where
     vote : ℚ → Maybe ℚ
     vote = just
 
--- TODO: this doesn't actually depend on PParams so we could remove that argument,
```

Comment:
Please don't just remove this TODO, this is still something that should be fixed